internal/test/test.go

```diff
@@ -0,0 +1,59 @@
+package test
+
+import (
+	"context"
+	"reflect"
+	"testing"
+)
+
+type Test interface {
+	Run(context.Context, *testing.T)
+}
+
+type test struct {
+	cs     []Caser
```

Comment:
I prefer `case` to `cs`, because `target` is not omitted.
please tell me your opinion.

Reply:
"case" is a reserved word in golang and can not be used. So omit the "target"? ?

Reply:
Thank you for your review.
Because `cs` type is `[]Caser`, I think `cases` is more appropriate than `case`
What do you think?